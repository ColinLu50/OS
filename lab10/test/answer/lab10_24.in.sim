47338
1217
1223
828
1217
821
