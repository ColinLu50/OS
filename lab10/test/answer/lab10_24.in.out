FCFS
395 -> 816 -> 70 -> 78 -> 71 -> 374 -> 377 -> 3 -> 702 -> 132 -> 764 -> 614 -> 195 -> 557 -> 706 -> 621 -> 67 -> 43 -> 677 -> 185 -> 476 -> 399 -> 373 -> 592 -> 760 -> 441 -> 582 -> 232 -> 538 -> 78 -> 44 -> 453 -> 28 -> 568 -> 443 -> 106 -> 676 -> 168 -> 59 -> 52 -> 598 -> 285 -> 624 -> 13 -> 161 -> 294 -> 389 -> 571 -> 98 -> 715 -> 684 -> 24 -> 18 -> 617 -> 391 -> 444 -> 499 -> 618 -> 680 -> 3 -> 93 -> 347 -> 454 -> 274 -> 678 -> 92 -> 642 -> 386 -> 288 -> 784 -> 162 -> 760 -> 383 -> 436 -> 174 -> 283 -> 248 -> 476 -> 227 -> 333 -> 757 -> 198 -> 498 -> 187 -> 395 -> 357 -> 298 -> 143 -> 591 -> 121 -> 828 -> 103 -> 89 -> 260 -> 801 -> 388 -> 139 -> 107 -> 569 -> 468 -> 392 -> 62 -> 520 -> 205 -> 435 -> 738 -> 240 -> 811 -> 293 -> 214 -> 418 -> 341 -> 224 -> 414 -> 665 -> 581 -> 529 -> 132 -> 654 -> 791 -> 223 -> 280 -> 42 -> 771 -> 17 -> 507 -> 745 -> 452 -> 519 -> 713 -> 762 -> 474 -> 91 -> 650 -> 343 -> 102 -> 785 -> 105 -> 287 -> 488 -> 788 -> 117 -> 458 -> 506 -> 187 -> 366 -> 695 -> 738 -> 635 -> 444 -> 826 -> 204 -> 313 -> 708 -> 431 -> 307 -> 366 -> 403 -> 360 -> 296 -> 249
47338
SSTF
395 -> 395 -> 392 -> 391 -> 389 -> 388 -> 386 -> 383 -> 377 -> 374 -> 373 -> 366 -> 366 -> 360 -> 357 -> 347 -> 343 -> 341 -> 333 -> 313 -> 307 -> 298 -> 296 -> 294 -> 293 -> 288 -> 287 -> 285 -> 283 -> 280 -> 274 -> 260 -> 249 -> 248 -> 240 -> 232 -> 227 -> 224 -> 223 -> 214 -> 205 -> 204 -> 198 -> 195 -> 187 -> 187 -> 185 -> 174 -> 168 -> 162 -> 161 -> 143 -> 139 -> 132 -> 132 -> 121 -> 117 -> 107 -> 106 -> 105 -> 103 -> 102 -> 98 -> 93 -> 92 -> 91 -> 89 -> 78 -> 78 -> 71 -> 70 -> 67 -> 62 -> 59 -> 52 -> 44 -> 43 -> 42 -> 28 -> 24 -> 18 -> 17 -> 13 -> 3 -> 3 -> 399 -> 403 -> 414 -> 418 -> 431 -> 435 -> 436 -> 441 -> 443 -> 444 -> 444 -> 452 -> 453 -> 454 -> 458 -> 468 -> 474 -> 476 -> 476 -> 488 -> 498 -> 499 -> 506 -> 507 -> 519 -> 520 -> 529 -> 538 -> 557 -> 568 -> 569 -> 571 -> 581 -> 582 -> 591 -> 592 -> 598 -> 614 -> 617 -> 618 -> 621 -> 624 -> 635 -> 642 -> 650 -> 654 -> 665 -> 676 -> 677 -> 678 -> 680 -> 684 -> 695 -> 702 -> 706 -> 708 -> 713 -> 715 -> 738 -> 738 -> 745 -> 757 -> 760 -> 760 -> 762 -> 764 -> 771 -> 784 -> 785 -> 788 -> 791 -> 801 -> 811 -> 816 -> 826 -> 828
1217
SCAN
395 -> 392 -> 391 -> 389 -> 388 -> 386 -> 383 -> 377 -> 374 -> 373 -> 366 -> 360 -> 357 -> 347 -> 343 -> 341 -> 333 -> 313 -> 307 -> 298 -> 296 -> 294 -> 293 -> 288 -> 287 -> 285 -> 283 -> 280 -> 274 -> 260 -> 249 -> 248 -> 240 -> 232 -> 227 -> 224 -> 223 -> 214 -> 205 -> 204 -> 198 -> 195 -> 187 -> 185 -> 174 -> 168 -> 162 -> 161 -> 143 -> 139 -> 132 -> 121 -> 117 -> 107 -> 106 -> 105 -> 103 -> 102 -> 98 -> 93 -> 92 -> 91 -> 89 -> 78 -> 71 -> 70 -> 67 -> 62 -> 59 -> 52 -> 44 -> 43 -> 42 -> 28 -> 24 -> 18 -> 17 -> 13 -> 3 -> 0 -> 399 -> 403 -> 414 -> 418 -> 431 -> 435 -> 436 -> 441 -> 443 -> 444 -> 452 -> 453 -> 454 -> 458 -> 468 -> 474 -> 476 -> 488 -> 498 -> 499 -> 506 -> 507 -> 519 -> 520 -> 529 -> 538 -> 557 -> 568 -> 569 -> 571 -> 581 -> 582 -> 591 -> 592 -> 598 -> 614 -> 617 -> 618 -> 621 -> 624 -> 635 -> 642 -> 650 -> 654 -> 665 -> 676 -> 677 -> 678 -> 680 -> 684 -> 695 -> 702 -> 706 -> 708 -> 713 -> 715 -> 738 -> 745 -> 757 -> 760 -> 762 -> 764 -> 771 -> 784 -> 785 -> 788 -> 791 -> 801 -> 811 -> 816 -> 826 -> 828
1223
C-SCAN
395 -> 392 -> 391 -> 389 -> 388 -> 386 -> 383 -> 377 -> 374 -> 373 -> 366 -> 360 -> 357 -> 347 -> 343 -> 341 -> 333 -> 313 -> 307 -> 298 -> 296 -> 294 -> 293 -> 288 -> 287 -> 285 -> 283 -> 280 -> 274 -> 260 -> 249 -> 248 -> 240 -> 232 -> 227 -> 224 -> 223 -> 214 -> 205 -> 204 -> 198 -> 195 -> 187 -> 185 -> 174 -> 168 -> 162 -> 161 -> 143 -> 139 -> 132 -> 121 -> 117 -> 107 -> 106 -> 105 -> 103 -> 102 -> 98 -> 93 -> 92 -> 91 -> 89 -> 78 -> 71 -> 70 -> 67 -> 62 -> 59 -> 52 -> 44 -> 43 -> 42 -> 28 -> 24 -> 18 -> 17 -> 13 -> 3 -> 0 -> 832 -> 828 -> 826 -> 816 -> 811 -> 801 -> 791 -> 788 -> 785 -> 784 -> 771 -> 764 -> 762 -> 760 -> 757 -> 745 -> 738 -> 715 -> 713 -> 708 -> 706 -> 702 -> 695 -> 684 -> 680 -> 678 -> 677 -> 676 -> 665 -> 654 -> 650 -> 642 -> 635 -> 624 -> 621 -> 618 -> 617 -> 614 -> 598 -> 592 -> 591 -> 582 -> 581 -> 571 -> 569 -> 568 -> 557 -> 538 -> 529 -> 520 -> 519 -> 507 -> 506 -> 499 -> 498 -> 488 -> 476 -> 474 -> 468 -> 458 -> 454 -> 453 -> 452 -> 444 -> 443 -> 441 -> 436 -> 435 -> 431 -> 418 -> 414 -> 403 -> 399
828
LOOK
395 -> 392 -> 391 -> 389 -> 388 -> 386 -> 383 -> 377 -> 374 -> 373 -> 366 -> 360 -> 357 -> 347 -> 343 -> 341 -> 333 -> 313 -> 307 -> 298 -> 296 -> 294 -> 293 -> 288 -> 287 -> 285 -> 283 -> 280 -> 274 -> 260 -> 249 -> 248 -> 240 -> 232 -> 227 -> 224 -> 223 -> 214 -> 205 -> 204 -> 198 -> 195 -> 187 -> 185 -> 174 -> 168 -> 162 -> 161 -> 143 -> 139 -> 132 -> 121 -> 117 -> 107 -> 106 -> 105 -> 103 -> 102 -> 98 -> 93 -> 92 -> 91 -> 89 -> 78 -> 71 -> 70 -> 67 -> 62 -> 59 -> 52 -> 44 -> 43 -> 42 -> 28 -> 24 -> 18 -> 17 -> 13 -> 3 -> 399 -> 403 -> 414 -> 418 -> 431 -> 435 -> 436 -> 441 -> 443 -> 444 -> 452 -> 453 -> 454 -> 458 -> 468 -> 474 -> 476 -> 488 -> 498 -> 499 -> 506 -> 507 -> 519 -> 520 -> 529 -> 538 -> 557 -> 568 -> 569 -> 571 -> 581 -> 582 -> 591 -> 592 -> 598 -> 614 -> 617 -> 618 -> 621 -> 624 -> 635 -> 642 -> 650 -> 654 -> 665 -> 676 -> 677 -> 678 -> 680 -> 684 -> 695 -> 702 -> 706 -> 708 -> 713 -> 715 -> 738 -> 745 -> 757 -> 760 -> 762 -> 764 -> 771 -> 784 -> 785 -> 788 -> 791 -> 801 -> 811 -> 816 -> 826 -> 828
1217
C-LOOK
395 -> 392 -> 391 -> 389 -> 388 -> 386 -> 383 -> 377 -> 374 -> 373 -> 366 -> 360 -> 357 -> 347 -> 343 -> 341 -> 333 -> 313 -> 307 -> 298 -> 296 -> 294 -> 293 -> 288 -> 287 -> 285 -> 283 -> 280 -> 274 -> 260 -> 249 -> 248 -> 240 -> 232 -> 227 -> 224 -> 223 -> 214 -> 205 -> 204 -> 198 -> 195 -> 187 -> 185 -> 174 -> 168 -> 162 -> 161 -> 143 -> 139 -> 132 -> 121 -> 117 -> 107 -> 106 -> 105 -> 103 -> 102 -> 98 -> 93 -> 92 -> 91 -> 89 -> 78 -> 71 -> 70 -> 67 -> 62 -> 59 -> 52 -> 44 -> 43 -> 42 -> 28 -> 24 -> 18 -> 17 -> 13 -> 3 -> 828 -> 826 -> 816 -> 811 -> 801 -> 791 -> 788 -> 785 -> 784 -> 771 -> 764 -> 762 -> 760 -> 757 -> 745 -> 738 -> 715 -> 713 -> 708 -> 706 -> 702 -> 695 -> 684 -> 680 -> 678 -> 677 -> 676 -> 665 -> 654 -> 650 -> 642 -> 635 -> 624 -> 621 -> 618 -> 617 -> 614 -> 598 -> 592 -> 591 -> 582 -> 581 -> 571 -> 569 -> 568 -> 557 -> 538 -> 529 -> 520 -> 519 -> 507 -> 506 -> 499 -> 498 -> 488 -> 476 -> 474 -> 468 -> 458 -> 454 -> 453 -> 452 -> 444 -> 443 -> 441 -> 436 -> 435 -> 431 -> 418 -> 414 -> 403 -> 399
821
